README.md

```diff
@@ -93,6 +93,20 @@ Run `make` in the root directory to compile the sources. For development, use
 `make debug` to build a non-optimized debug version, or `make release` to build
 an optimized version.  Use `make unittest` to build and run tests.
 
+Note that,
+* Velox requires C++17 , thus minimum supported compiler is GCC 5.0 and Clang 5.0.
+* Velox requires the CPU to support instruction sets:
+  * bmi
```

Comment:
My compilation failed without bmi/bmi2 even with Jimmys latest changes, so these are still required.

Reply:
We will have some recommended flags which will include BMI2 and AVX2, and another set of minimal flags (e.g. SSE4.2).